### PR TITLE
chore: bump sso-express version

### PR DESCRIPTION
**Pull request by BryanGK**
This PR bumps the sso-express version to 2.1.0

### CODE REVIEW

**Review by wenzowski — APPROVED** [review]

**Comment by wenzowski** [comment]
Sounds like we’re folding migration from public to confidential client into this PR; that change will require re-approval

File: helm/templates/ssoSecret.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: Secret
+metadata:
+  name: sso-express
+  labels: 
+{{ include "connectivity-intake.labels" . | indent 4}}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
+type: Opaque
+data:
+  clientSecret: {{ .Values.image.app.clientSecret }}
```
Comment (wenzowski) [inline helm/templates/ssoSecret.yaml:11]:
```suggestion
  clientSecret: {{ .Values.image.app.clientSecret  | b64enc | quote }}
```

**Review by wenzowski — CHANGES_REQUESTED** [review]
Did you forget to encode your secret?

File: helm/templates/ssoSecret.yaml (hunk 1 above)
Comment (BryanGK) [reply helm/templates/ssoSecret.yaml]: yeah just pushed a change to test deployment

File: helm/templates/deploy.yaml [hunk 2]
```diff
@@ -154,6 +154,11 @@ spec:
               secretKeyRef:
                 key: session-secret
                 name: {{ template "connectivity-intake.fullname" . }}
+          - name: CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                key: client-secret
+                name: {{ template "connectivity-intake.fullname" . }}-client-secret
```
Comment (wenzowski) [inline helm/templates/deploy.yaml:161]:
```suggestion
                name: {{ template "connectivity-intake.fullname" . }}-sso-express
```

File: helm/templates/ssoSecret.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sso-express
```
Comment (wenzowski) [inline helm/templates/ssoSecret.yaml:4]:
```suggestion
  name: {{ template "connectivity-intake.fullname" . }}-sso-express
```

**Review by wenzowski — CHANGES_REQUESTED** [review]
secret name doesn't match

**Review by wenzowski — APPROVED** [review]

**Pull request merged**